refresh/templates/common/README.scaffold.md

```diff
@@ -66,15 +66,20 @@ To build and run the application:
 
 {{#if docker}}
 #### Docker
-A description of the files related to Docker can be found in the [Docker files](#docker-files) setion. To build the two docker images, run the following commands from the root directory of the project:
-* `docker build -t myapp-run .`
+A description of the files related to Docker can be found in the [Docker files](#docker-files) section. To build the two docker images, run the following commands from the root directory of the project.
+
+First, build the 'tools' image. This image contains a full Linux Swift toolchain and is capable of compiling your application for Linux:
```

Comment:
Might be worth explaining that `-t` tags the image. Or specify `--tag` instead of `-t`.